### ISSUE TITLE
Errors when using opbeat with got@7

### ISSUE THREAD

**Issue by MartinKolarik** [opbeat/opbeat-node#179]
Opbeat's HTTP instrumentation breaks something in [got@7](https://www.npmjs.com/package/got) and causes random HTTP related errors. The code below should be enough to reproduce `Error: write after end` when visiting the webpage:

```js
// opbeat setup here
// ...

const express = require('express');
const got = require('got');
const app = express();

app.use('/', (req, res) => {
	got('https://google.com').then(() => {
		res.send('Hello World!');
	});
});

app.listen(7777);
```

**Comment by watson** [comment]
Thanks for reporting this issue. We're working to release a fix ASAP

**Comment by watson** [comment]
@MartinKolarik this have now been fixed in version 4.15.1

**Comment by MartinKolarik** [comment]
I can confirm it is fixed. Thanks @watson !
